index.html

```diff
@@ -0,0 +1,170 @@
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/CC.svg" alt="CC Excibition">
+                    <h2>CC Exhibition</h2>
+                    <p>Appreciate various creations applying CC license of artists, organized from art center Nabi.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/forum.svg" alt="Forum">
+                    <h2>Forum</h2>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/workshop.svg" alt="Workshop">
+                    <h2>Workshop</h2>
+                    <p>Try creating your own work using open source license rather than just wathcing</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/ignite.svg" alt="CC Ignite">
+                    <h2>CC Ignite</h2>
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+            <a href="#">see the whole Program</a>
+            <button class="join">Join CC Global Summit 2015</button>
+        </div>
+        </div>
+
+        <div class="featured">
+            <div class="container">
+            <h1>Featured Speakers</h1>
+            <div class="featured-items">
+
+                <div class="featured-speakers">                    
+                </div>
+                <button class="more">more</button>
+                <div class="featured-speakers2"></div>
+                
+
+            </div>
+        </div>
+            <div class="partner">
+                <div class="container">
+                    <h6>Partner</h6>
+                <div class="grid3">
+                    <a href="#">mozilla</a>
+                    <a href="#">Google</a>
+                    <a href="#">Naver</a>
+                    <a href="#">daumkakao</a>
+                    <a href="#">airbnb</a>
+                </div>
+                </div>
+                
+
+                <footer class="main-footer">
+                    <div class="container">
+                        <div class="footer">
+                            <img src="images/cck-logo1 1.svg" alt="logo">
+                    <div class="foter-para">
+                        <p class="p1">2015 Creative Commons Korea. Some Rights Reserved.</p>
+                        <!-- <span></span> -->
+                        <p class="p2">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                            자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+                    </div>
+                        </div>
+                        
+                    </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add some spaces to the `up` and `down` parts of the footer section as mentioned by the [previous reviewer](https://github.com/abdozayan12/capstone-project---Conference-page/tree/Conference---page#r1203932997) 🙏🏼 

![image](https://github.com/abdozayan12/capstone-project---Conference-page/assets/63915024/231db1e6-81ec-491e-be28-9637f152c54a)